create.cjs

```diff
@@ -425,9 +461,29 @@ const manualPackagesList = /** @type {const} */ ([
     }
   };
 
+  const fetchOriginPackageSize = async (packageName) => {
+    const packageJson = await makeRequest('https://packagephobia.com/api.json', {
```

Comment:
Is it possible to use `registry.npmjs.org` instead? `registry.npmjs.org` is way more reliable.

E.g. You can get `unpackedSize` and `fileCount` from `https://registry.npmjs.org/nolyfill/latest`.

Reply:
It's possible but only if the dependency tree is evaluated correctly. The tree of yarn is incorrect currently, however.

Check the discussions in the tg group for details.